findAndReplacePassportValue/prompts.js

```diff
@@ -0,0 +1,22 @@
+const { setupPrompts } = require("../prompts");
+
+const findAndReplacePrompts= [
+  ...setupPrompts,
+  {
+    name: "currentPassportVariable",
+    description: "What is the current passport variable to be replaced?",
+    default: "outbuildings",
+    type: "string",
+    required: true,
+  },
+  {
+    name: "newPassportVariable",
+    description: "What should this passport variable be replaced with?",
+    default: "outbuilding",
+    type: "string",
+    required: true,
+  },
```

Comment:
:+1: similar to comment above - these are currently phrased as if I can pass a _full_ passport variable to replace (for example `alter.opening` &rarr; `alteration.openings`) 

But the logic in the replace function actually only works on _segments_ of variables. Would be great to be consistent and preference flexibility of replacing any number of characters independent of segments please!